src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingAssessmentIntegrationTest.java

```diff
@@ -602,7 +594,8 @@ private void assessmentDueDatePassed() {
     private void overrideAssessment(HttpStatus httpStatus) throws Exception {
         var participation = programmingSubmission.getParticipation();
         var result = programmingSubmission.getLatestResult();
-        result.setScore(75L);
+        assert result != null;
```

Comment:
```suggestion
        assertThat(result).isNotNull();
```
Because we use this formulation in other parts in this Test class